From 3d0480df19355b2a67333af45bf64b64cfdc9b51 Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Tue, 20 Dec 2011 10:02:29 +0800
Subject: ENGR00170548-1 [IMX6Q] USB-HOST:Enable the USB video and audio

Enable the USB video config as module avoid conflict with CSI camera
Enable the USB audio config as defult built in driver

Signed-off-by: make shi <b15407@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index c319c01..4b6496f 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1381,7 +1381,10 @@ CONFIG_VIDEO_MXC_IPU_OUTPUT=y
 # CONFIG_VIDEO_TIMBERDALE is not set
 # CONFIG_VIDEO_SR030PC30 is not set
 # CONFIG_SOC_CAMERA is not set
-# CONFIG_V4L_USB_DRIVERS is not set
+CONFIG_V4L_USB_DRIVERS=y
+CONFIG_USB_VIDEO_CLASS=m
+CONFIG_USB_VIDEO_CLASS_INPUT_EVDEV=y
+CONFIG_USB_VIDEO_BUFFERS_DMA=y
 # CONFIG_V4L_MEM2MEM_DRIVERS is not set
 # CONFIG_RADIO_ADAPTERS is not set
 
@@ -1500,7 +1503,7 @@ CONFIG_SND_DRIVERS=y
 CONFIG_SND_ARM=y
 CONFIG_SND_SPI=y
 CONFIG_SND_USB=y
-# CONFIG_SND_USB_AUDIO is not set
+CONFIG_SND_USB_AUDIO=y
 # CONFIG_SND_USB_UA101 is not set
 # CONFIG_SND_USB_CAIAQ is not set
 CONFIG_SND_SOC=y
-- 
1.7.9.5

